partition/roles/sonic/tasks/generate_config_db.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Create temporary directory
+  file:
+    path: /tmp/sonic
+    state: directory
+
+- name: Render input values template
+  template:
+    src: sonic-config.yaml.j2
+    dest: /tmp/sonic/sonic-config.yaml
+
+- name: Copy current config_db.json into temporary directory
+  copy:
+    src: /etc/sonic/config_db.json
+    dest: /tmp/sonic/config_db.json
+    mode: 0644
+    remote_src: true
```

Comment:
What happens if no `config_db.json` already exists?

Reply:
See above. `sudo config-setup factory` ensures there is one.